docs/docusaurus/docs/terms/execution_engine.md

```diff
@@ -18,7 +18,7 @@ Execution Engines are components of <TechnicalTag relative="../" tag="datasource
 
 ## Use cases
 
-An Execution Engine is defined in the configuration of a Datasource.  After this, you will not need to directly interact with an Execution Engine.  Instead, it will be employed under the hood by the Datasoruce it is configured for.
+An Execution Engine is defined in the configuration of a Datasource.  After this, you will not need to directly interact with an Execution Engine.  Instead, it will be employed under the hood by the Datasource it is configured for.
```

Comment:
```suggestion
You define the Execution Engine that you want to use to process data to compute Metrics in the Datasource configuration.  After you define the Execution Engine, you don't need to interact with it because the Datasource it is configured for uses it automatically.
```
Avoid metaphors (under the hood) in technical documentation. They are not easily understood by all readers.